xbmc/windowing/win10/WinEventsWin10.cpp

```diff
@@ -182,6 +185,11 @@ void CWinEventsWin10::InitEventHandlers(CoreWindow^ window)
     m_smtc->IsEnabled = true;
     CAnnouncementManager::GetInstance().AddAnnouncer(this);
   }
+  if (CSysInfo::GetWindowsDeviceFamily() == CSysInfo::WindowsDeviceFamily::Xbox)
+  {
+    m_remote = std::make_unique<CRemoteControlXbox>();
```

Comment:
`std::make_unique` is c++14, use `m_remote.reset(new CRemoteControlXbox);`

Reply:
this is windows code only. msvc supports some c++14 features.